plugins/statuslines/lualine.nix

```diff
@@ -80,6 +80,10 @@ with lib; let
               "1"
               "Adds padding to the left and right of components.";
 
+            fmt = helpers.mkNullOrOption helpers.nixvimTypes.rawLua ''
```

Comment:
I'd suggest making an option of type `nullOr strLuaFn` and using an `apply` to map it to a raw value, so the user can simply input a string (see other options that use `apply`, many of them do that)